### PR TITLE
Remove Usage of Var

### PR DESCRIPTION
**Pull request by Axionize**
I'm fine with the final which makes it clear a variable is never modified but not knowing a var type at a glance is terrible.

### CODE REVIEW

File: src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java [hunk 1]
```diff
@@ -453,8 +453,8 @@ public void onPacketReceive(PacketReceiveEvent event) {
         }
 
         if (event.getPacketType() == PacketType.Play.Client.PLAYER_DIGGING) {
-            final var packet = new WrapperPlayClientPlayerDigging(event);
-            final var blockBreak = new BlockBreak(packet, player);
+            final WrapperPlayClientPlayerDigging packet = new WrapperPlayClientPlayerDigging(event);
+            final BlockBreak blockBreak = new BlockBreak(packet, player);
```
Comment (ManInMyVan) [inline src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java:457]: I think it's pretty obvious what the type of these are, they use a constructor

**Review by ManInMyVan — CHANGES_REQUESTED** [review]

File: src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java (hunk 1 above)
Comment (Axionize) [reply src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java:457]: It's consistent with the rest of the code base, and while I can only speak for myself on this front, when I look for the type of something I always first look to the left then right

**Review by ManInMyVan — APPROVED** [review]

File: src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java (hunk 1 above)
Comment (SamB440) [reply src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java:457]: I don’t see a need to go reverting every var change, using var here makes sense as it shortens the line whilst the type is still clear

Comment (Axionize) [reply src/main/java/ac/grim/grimac/events/packets/CheckManagerListener.java:457]: @MachineBreaker thoughts?

**Comment by ghost** [comment]
W change, reading code on github with var in it is awful

**Pull request merged**